apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.js

```diff
@@ -35,7 +34,7 @@ router.get('/:job_id', (req, res, next) => {
     if (result.rows[0].aai && !result.rows[0].aai.authorized) {
       return next(error.make(403, 'Access denied (must be a student data viewer)'));
     }
```

Comment:
For other reviewers: we should never expect to get data for `authz_assessment_instance` for the course page, as any grading job associated only with a course must by definition not have an associated assessment instance.